src/components/_global/BalTable/BalTable.vue

```diff
@@ -36,7 +36,7 @@
               ></slot>
               <div v-else>
                 <h5
-                  class="text-base font-semibold text-gray-800 dark:text-gray-100"
+                  class="text-base text-gray-800 dark:text-gray-100"
```

Comment:
Please fix lint issue.

Reply:
If you scroll threw the [files view ](https://github.com/balancer-labs/frontend-v2/pull/558/files) of the PR you'll see this big warning:
<img width="1380" alt="Screenshot 2021-07-13 at 19 39 23" src="https://user-images.githubusercontent.com/2406506/125507233-78a9a600-fab3-4b79-8898-8fa35a6b31e1.png">

Reply:
This should be fixed